TLM/TLM/Custom/PathFinding/CustomPathFind.cs

```diff
@@ -200,37 +161,47 @@ private struct BufferItem {
             Type stockPathFindType = typeof(PathFind);
             const BindingFlags FIELD_FLAGS = BindingFlags.NonPublic | BindingFlags.Instance;
 
-            pathUnitsField_ = stockPathFindType.GetField("m_pathUnits", FIELD_FLAGS);
-            queueFirstField_ = stockPathFindType.GetField("m_queueFirst", FIELD_FLAGS);
-            queueLastField_ = stockPathFindType.GetField("m_queueLast", FIELD_FLAGS);
-            queueLockField_ = stockPathFindType.GetField("m_queueLock", FIELD_FLAGS);
-            terminatedField_ = stockPathFindType.GetField("m_terminated", FIELD_FLAGS);
-            calculatingField_ = stockPathFindType.GetField("m_calculating", FIELD_FLAGS);
-            pathFindThreadField_ = stockPathFindType.GetField("m_pathFindThread", FIELD_FLAGS);
+            queueFirst_ = 0;
```

Comment:
hopefully avoiding the initialization does not cause any hot-reload issues.

Reply:
None of `CustomPathFind` functions use those fields, neither game use those, and the hot-reload rollback is instantiating original classes so no risk of leak or something like that 😉